proto/ligato/vpp/srv6/srv6.proto

```diff
@@ -132,3 +132,8 @@ message Steering {
         string prefix_address = 2;       /* IPv4/IPv6 prefix address(CIRD format) of traffic destination. All traffic with given destination will be steered to given SR policy */
     }
 }
+
+/* IPv6 SR Set SRv6 encapsulation source */
+message EncapSourceAddress {
```

Comment:
I think that `EncapSourceAddress` message should rather be named in a way that it can be extended in the future when possibly some other _srv6 related_ global settings is needed. Perhaps `SRv6Global`?  What do you think? @artem-belov

Reply:
I agree